apps/dashboard-service/src/dashboard/dto/total-serction-header.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { JsonApiDto } from "@terramatch-microservices/common/decorators";
+import { ApiProperty } from "@nestjs/swagger";
+import { populateDto } from "@terramatch-microservices/common/dto/json-api-attributes";
+
+@JsonApiDto({ type: "totalSectionHeaders" })
+export class TotalSectionHeaderDto {
```

Comment:
Please update the filename to match the class (`section` is misspelled in the filename)